depmapomics/mutations.py

```diff
@@ -657,6 +689,5 @@ def postprocess_main_steps(maf: pd.DataFrame, adjusted_gnomad_af_cutoff: float=1
         internal_afs_ratio_dict[k] = v / total_samples
     maf.loc[:, "internal_afs"] = internal_afs.map(internal_afs_ratio_dict)
     maf = maf.loc[(maf.internal_afs <= max_recurrence) | (maf.rescue), :]
-    maf['version'] = version
```

Comment:
@qinqian I'm not sure how useful the "version" column is here actually. The release number should be apparent to the users anyways

Reply:
I agree, then we could remove this line